vtex-apps/store-image
# Fix/image title attr

#### What problem is this solving?


It was lefting some steps to finish adding the title attribute to Image/ImageList/ImageSlider components in the image/link anchors

#### How to test it?

You can go to CMS -> SiteEditor -> ImageList and change the title attribute of the images/links. Now, in the worskpace, you will have to inspect image/link tag in the ImageList section(the last) to check if the attribute was updated!

https://vitorflg--storecomponents.myvtex.com/admin/cms/site-editor
https://vitorflg--storecomponents.myvtex.com/

#### How does this PR make you feel? [:link:](https://media.giphy.com/media/l4KhPbIIDgO3sMw0w/giphy.gif)

![Image](https://media.giphy.com/media/l4KhPbIIDgO3sMw0w/giphy.gif)
